app/Http/Controllers/Api/ReviewsController.php

```diff
@@ -43,6 +43,8 @@ class ReviewsController extends ApiController
      */
     public function __construct(PostService $posts, ReportbackService $reportbacks)
     {
+        $this->middleware('api');
```

Comment:
Sorry if I missed this elsewhere, but what's the rationale for moving the `api` middleware group onto the individual controllers? It seems like it makes it a bit easier to forget.